website/content/en/preview/upgrade-guide/_index.md

```diff
@@ -70,7 +70,8 @@ for a subset of older versions and deprecate the others.
 # Released Upgrade Notes
 
 ## Upgrading to v0.13.0+
-v0.13.0 adds EC2/spot price fetching to Karpenter to allow making more accurate decisions regarding node deployments.  Our getting started guide documents this, but if you are upgrading Karpenter you will need to modify your Karpenter controller policy to add the `pricing:GetProducts` and `ec2:DescribeSpotPriceHistory` permissions.    
+v0.13.0 introduces a new CRD named `AWSNodeTemplate` which can be used to specify AWS Cloud Provider parameters. Everything that was previously specified under `spec.provider` in the Provisioner resource, can now be specified in the spec of the new resource. The use of `spec.provider` is deprecated but will continue to function to maintain backwards compatibility for the current API version (v1alpha5) of the Provisioner resource. v0.13.0 also introduces support for custom user data that doesn't require the use of a custom launch template. The user data can be specified in-line in the AWSNodeTemplate resource. Read the [UserData documentation here](../AWS/user-data.md) to get started.
```

Comment:
I think the link to user-data is broken.  Think it should just be `../aws/user-data/`.

Reply:
I also noticed some strangeness when installing the helm chart with the AWSNodeTemplate CRD.  I believe we renamed it from `awsnodetemplate.karpenter.k8s.aws` to `awsnodetemplates.karpenter.k8s.aws` at some point, which I think was the cause.

Reply:
* Yup, let me fix the link. 
* Lmk what was going wrong. Helm does make it really difficult to manage CRDs.